packages/canary-react/src/stories/ic-data-table.stories.mdx

```diff
@@ -1088,36 +1088,38 @@ const DataTable = () => {
 export default DataTable;
 ```
 
-### Links and elements in data
+### Links and slotted elements in data
```

Comment:
There are still some "Links and elements in data" instances in this file (lines 1098, 1129 and 1169) - would be good to update those to include the "slotted" to match